publishing failing on phoenix server, API test suite down

On 860d53e I tried to run the phoenix tests but the failed because publishing isn't working.

![Screen Shot 2019-09-16 at 4 45 23 PM](https://user-images.githubusercontent.com/21006/64992301-9acd7080-d8a1-11e9-81b2-4d6d70f04e93.png)


Caused by: java.lang.RuntimeException: Response code: 403, Access is denied
	at edu.harvard.iq.dataverse.DataCiteRESTfullClient.postMetadata(DataCiteRESTfullClient.java:190)
	at edu.harvard.iq.dataverse.DOIDataCiteRegisterService.registerIdentifier(DOIDataCiteRegisterService.java:110)

[2019-09-16T16:44:16.259-0400] [glassfish 4.1] [SEVERE] [] [edu.harvard.iq.dataverse.DatasetPage] [tid: _ThreadID=50 _ThreadName=jk-connector(1)] [timeMillis: 1568666656259] [levelValue: 1000] [[
  This dataset may not be published due to an error when contacting the <a href=http://status.datacite.org target="_blank"/> DataCite </a> Service. Please try again.]]

![Screen Shot 2019-09-16 at 4 45 20 PM](https://user-images.githubusercontent.com/21006/64992300-9acd7080-d8a1-11e9-99b4-fe52e83818b1.png)

--- a/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java
+++ b/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java
@@ -518,7 +518,22 @@ public class SettingsServiceBean {
     }
      
     public Setting set( String name, String content ) {
-        Setting s = new Setting( name, content );
+        Setting s = null; 
+        
+        List<Setting> tokens = em.createNamedQuery("Setting.findByName", Setting.class)
+                .setParameter("name", name )
+                .getResultList();
+        
+        if(tokens.size() > 0) {
+            s = tokens.get(0);
+        }
+        
+        if (s == null) {
+            s = new Setting( name, content );
+        } else {
+            s.setContent(content);
+        }
+        
         s = em.merge(s);
         actionLogSvc.log( new ActionLogRecord(ActionLogRecord.ActionType.Setting, "set")
                             .setInfo(name + ": " + content));
@@ -526,7 +541,23 @@ public class SettingsServiceBean {
     }
 
     public Setting set( String name, String lang, String content ) {
-        Setting s = new Setting( name, lang, content );
+        Setting s = null; 
+        
+        List<Setting> tokens = em.createNamedQuery("Setting.findByNameAndLang", Setting.class)
+                .setParameter("name", name )
+                .setParameter("lang", lang )
+                .getResultList();
+        
+        if(tokens.size() > 0) {
+            s = tokens.get(0);
+        }
+        
+        if (s == null) {
+            s = new Setting( name, lang, content );
+        } else {
+            s.setContent(content);
+        }
+        
         em.merge(s);
         actionLogSvc.log( new ActionLogRecord(ActionLogRecord.ActionType.Setting, "set")
                 .setInfo(name + ": " +lang + ": " + content));

